log4j-api-test/src/main/java/org/apache/logging/log4j/test/junit/package-info.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the license.
  */
 @Export
-@Version("2.21.1")
+@Version("2.22.0")
```

Comment:
I thought we were sticking to `bnd-baseline-maven-plugin` suggestions?

> keep the OSGi version somehow related to the artifact version

I thought these two are almost unrelated. Am I missing something?

Reply:
You might have missed my proposal: https://lists.apache.org/thread/tv3t11pwq54ch56qrj6odz9hngys1761

Rationale: it might be confusing to users to match the package version with the artifact version. If they see "2.22.0", they might think `log4j-api` 2.22.0 is enough. So I would propose:

- for a micro bump, just increase the micro number (e.g. `2.21.1` -> `2.21.2`). If users compiled their code against package number `2.21.7`, it should still work with `log4j-api` 2.21.0 (module bug fixes),
- for a minor bump, align the version number with the version of `log4j-api`. If the package is `2.21.1` we bump it to `2.23.0` in 2.x and `3.0.0` in 3.x (unless we can guarantee that the changes in 2.x and 3.x are synchronized).

Reply:
If we use the module version as the package version, then we should also update the various packages marked as version `1.0.0`. And if `Foo.class.getPackage().getSpecificationVersion()` (and `getImplementationVersion()`) return the module version rather than the OSGi package version, that's just further confusion!

Reply:
Fixed in 2c01bcd2d94a4f5a20daa2b04181b939a25b83db.